### PR TITLE
Moved veggieseasons out of experimental

### PR DESCRIPTION
**Pull request by tusharojha**
## Description

As state restoration lands to the stable with Flutter 2. I have moved the Veggie Seasons sample out of experimental. 

Fixes: #749 

@domesticmouse @RedBrogdon PTAL.

### CODE REVIEW

File: experimental/veggieseasons/README.md [hunk 1]
```diff
@@ -1,42 +0,0 @@
-# Veggie Seasons
```
Comment (domesticmouse) [inline experimental/veggieseasons/README.md:1]: Can you add this file back with a pointer to the new location? This way anyone linked to this directory will see an appropriate redirection notice.

**Review by domesticmouse — APPROVED** [review]
LGTM, once a README.md is added to `experimental/veggieseasons` pointing users back to the original directory

File: experimental/veggieseasons/README.md (hunk 1 above)
Comment (tusharojha) [reply experimental/veggieseasons/README.md:1]: Sure, I have updated the README accordingly. PTAL.

File: tool/flutter_ci_script_beta.sh [hunk 2]
```diff
@@ -30,7 +30,6 @@ declare -ar PROJECT_NAMES=(
     # "experimental/federated_plugin/federated_plugin" \
     "null_safety/null_safe_app" \
     "null_safety/null_unsafe_app" \
-    "experimental/veggieseasons" \
```
Comment (domesticmouse) [inline tool/flutter_ci_script_beta.sh:33]: Add veggieseasons back. It needs to be tested on all three branches.

File: tool/flutter_ci_script_dev.sh [hunk 3]
```diff
@@ -32,7 +32,6 @@ declare -ar PROJECT_NAMES=(
     # "experimental/federated_plugin/federated_plugin" \
     "null_safety/null_safe_app" \
     "null_safety/null_unsafe_app" \
-    "experimental/veggieseasons" \
```
Comment (domesticmouse) [inline tool/flutter_ci_script_dev.sh:35]: Likewise, add veggiesasons

**Review by domesticmouse — APPROVED** [review]

**Comment by domesticmouse** [comment]
Thanks @tusharojha!

**Pull request merged**